_data/sites.json

```diff
@@ -9243,6 +9243,19 @@
         ]
     },
 
+     {
+        "name": "Shein",
+        "url": "https://shein.com/robot",
+        "difficulty": "hard",
+        "notes": "Click on the link, then click on "Contact Agent", select"Unsubscribe and delete". Finally fill in the various fields and then press "Submit".
+⚠️ It is imperative to give a reason for deleting an account for it to work.",
```

Comment:
```suggestion
        "notes": "Click on the link, then click on \"Contact Agent\", select \"Unsubscribe and delete\". Finally fill in the various fields and then press \"Submit\". ⚠️ It is imperative to give a reason for deleting an account for it to work.",
```

We have to escape the quotes, otherwise the JSON will become malformed. As for the emoji, no idea whether it will work or not, I'll test locally.